internal/utils/csi.go

```diff
@@ -337,10 +337,13 @@ var DriverNamespaceEnvVar = corev1.EnvVar{
 	},
 }
 
-// CSI Addons container port definition
-var CsiAddonsContainerPort = corev1.ContainerPort{
+// CSI Addons container port definitions
+var CsiAddonsContainerPort1 = corev1.ContainerPort{
```

Comment:
I have few question regarding the port in use here @Madhu-1 
1. What is the use of this port?
2. Is this port information required anywhere else?
3. Since we know that we need to mitigate port collision when we use hostNetwork, can we hard code new ports?

Reply:
the ports are in faxt hard-coded in `internal/utils/csi.go`

Reply:
> I have few question regarding the port in use here @Madhu-1
> 
> 1. What is the use of this port?
> 2. Is this port information required anywhere else?

These ports are used for csi-addons server

> 3. Since we know that we need to mitigate port collision when we use hostNetwork, can we hard code new ports?

Yes, it can be hardcoded in this PR but we need to have a new PR to make it configurable as we support creating multiple instances of the csi drives. For 2 instances of driver will get in to the same problem of port (This bug we have today as well for daemonset)

Reply:
@Madhu-1 : do you mean a PR as preparation for this one?